library/src/main/java/com/novoda/viewpageradapter/ViewPagerAdapterState.java

```diff
@@ -40,8 +40,11 @@ private static ViewPagerAdapterState from(Parcel in) {
     }
 
     private static SparseIntArray extractIdsFrom(Bundle bundle) {
-        SparseIntArray output = new SparseIntArray();
         int[] ids = bundle.getIntArray(KEY_VIEW_IDS);
+        if (ids == null) {
```

Comment:
Fair points, but we control that bundle, see `writeToParcel()`. However, I don't see any harm in the check, so we're fine 😄